src/pages/bug-bounty.tsx

```diff
@@ -418,7 +417,7 @@ const BugBountiesPage = () => {
       link: "https://pegasys.tech/teku",
       image: useColorModeValue(tekuDark, tekuLight),
     },
-  ].map((client) => ({ ...client, image: resizeImage(client.image, 24) }))
+  ].map((client) => ({ ...client, image: client.image }))
```

Comment:
no need for the map, right?
```suggestion
  ]
```